src/web/mjs/connectors/Team1x1.mjs

```diff
@@ -7,56 +7,69 @@ export default class Team1x1 extends Connector {
-        let id = uri.pathname + uri.search;
-        let title = data[0].textContent;
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'div.row > div.col-md-9 > h3');
+        const id = uri.pathname + uri.search;
+        const title = data[0].textContent;
         return new Manga(this, id, title);
     }
 
-    async _getMangaListFromPages( mangaPageLinks, index ) {
+    async _getMangaListFromPages(mangaPageLinks, index) {
         index = index || 0;
-        let request = new Request( mangaPageLinks[ index ], this.requestOptions );
-        let data = await this.fetchDOM( request, 'div#page-manga div.container div.last-post-manga div.thumb div.info h3 a', 5 );
-        let mangaList = data.map( element => {
+        const request = new Request(mangaPageLinks[index], this.requestOptions);
+        const data = await this.fetchDOM(request, 'div#page-manga div.container div.last-post-manga div.thumb div.info h3 a', 5);
+        let mangaList = data.map(element => {
             return {
-                id: this.getRootRelativeOrAbsoluteLink( element, request.url ),
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
                 title: element.text.trim()
             };
-        } );
-        if( index < mangaPageLinks.length - 1 ) {
-            return this._getMangaListFromPages( mangaPageLinks, index + 1 )
-                .then( mangas => mangaList.concat( mangas ) );
+        });
+        if (index < mangaPageLinks.length - 1) {
+            return this._getMangaListFromPages(mangaPageLinks, index + 1)
+                .then(mangas => mangaList.concat(mangas));
         } else {
-            return Promise.resolve( mangaList );
+            return Promise.resolve(mangaList);
         }
     }
 
     async _getMangas() {
-        let request = new Request( this.url + '/manga/', this.requestOptions );
-        let data = await this.fetchDOM( request, 'div.pagination div.wp-pagenavi > a.last' );
-        let pageCount = parseInt( data[0].href.split("/").slice(-2, -1)[0].trim() );
-        let pageLinks = [... new Array( pageCount ).keys()].map( page => request.url + 'page/' + ( page + 1 ) + '/' );
-        return await this._getMangaListFromPages( pageLinks );
+        const request = new Request(this.url + '/manga/', this.requestOptions);
+        const data = await this.fetchDOM(request, 'div.pagination div.wp-pagenavi > a.last');
+        const pageCount = parseInt(data[0].href.split("/").slice(-2, -1)[0].trim());
+        const pageLinks = [... new Array(pageCount).keys()].map(page => request.url + 'page/' + (page + 1) + '/');
+        return await this._getMangaListFromPages(pageLinks);
+    }
+
+    async _getChapters(manga) {
+        let chapterList = [];
+        const request = new Request(this.url + manga.id, this.requestOptions);
+        const data = await this.fetchDOM(request, 'span.current');
```

Comment:
May improve query to make it more robust `div. pagination span.current`.
Is `span.current` really the last element? Sounds to me like it is the current page (which is probably always `1`).

Reply:
OK, seems the website automatically sets the last page as the current, e.g.
https://tqneplus.com/manga/the-heavens-list/

![image](https://user-images.githubusercontent.com/27136773/101872205-273e1000-3b85-11eb-8e61-b123add07dc1.png)

Reply:
yea, as u said it is the current page and that is the last page, i think it's an arabic left to right thing